playground/ci/test_grpc_client.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+from unittest.mock import AsyncMock
+import pytest
+
+from ci.api import api_pb2
+from ci.grpc_client import GRPCClient
+
+
+class TestGRPCClient:
+
+    @pytest.fixture()
+    def mock_run_code(self, mocker):
+        async_mock = AsyncMock()
+        mocker.patch('ci.grpc_client.GRPCClient.run_code', side_effect=async_mock)
```

Comment:
You should add return value here instead of adding it to the test itself. If you want to remember generated UUID, you can also create a fixture out of it